local.tf

```
@@ -19,10 +19,10 @@ locals {
     asg_max_size                  = "3"                             # Maximum worker capacity in the autoscaling group.
     asg_min_size                  = "1"                             # Minimum worker capacity in the autoscaling group.
     asg_force_delete              = false                           # Enable forced deletion for the autoscaling group.
-    instance_type                 = "m4.large"                      # Size of the workers instances.
+    instance_type                 = "m4.2xlarge"                      # Size of the workers instances.
```

We should keep a default instance type, but move to m5 since they're faster and cheaper
```suggestion
    instance_type                 = "m5.2xlarge"                      # Size of the workers instances.
```

---

👍